tuna/miopen/subcmd/load_job.py

```diff
@@ -207,15 +151,15 @@ def add_jobs(args, dbt):
     res = query.all()
 
     if not res:
-      LOGGER.error('No applicable solvers found for args %s', args.__dict__)
+      logger.error('No applicable solvers found for args %s', args.__dict__)
 
     fin_step_str = 'not_fin'
     if args.fin_steps:
       fin_step_str = ','.join(args.fin_steps)
     query = f"select config, solver from {dbt.job_table.__tablename__} where session={args.session_id} and fin_step='{fin_step_str}' and reason='{args.label}'"
-    LOGGER.info(query)
+    logger.info(query)
     ret = session.execute(query)
-    pre_ex = {}
+    pre_ex: Dict[str, Dict[str, bool]] = {}
```

Comment:
is the value in this dict str: bool or str: int. Im thinking it would map solver name to solver_id. @cderb  can you confirm?

Reply:
If we observe this line: pre_ex[config][solver] = True
I would say it is a Dict[int, Dict[str, bool]], as config is an id and solver is a solver namestring. Though it isn't a huge deal if config is compared here as a string.